runtime/near-vm-logic/src/dependencies.rs

```diff
@@ -486,4 +531,11 @@ pub trait External {
 
     /// Returns total stake of validators in the current epoch.
     fn validator_total_stake(&self) -> Result<Balance>;
+
+    /// Distribute the gas among the scheduled function calls that specify a gas ratio.
+    /// Returns the amount of distributed gas.
+    ///
+    /// * `gas` - amount of unused gas to distribute
```

Comment:
Please describe the returned value.

Reply:
The returned value will be removed based on the following comment, as all gas will be distributed.